src/cgr_gwas_qc/workflow/scripts/vcf2adpc.py

```diff
@@ -27,7 +27,7 @@ def main(
     vcf_file: Path = typer.Argument(
         ..., help="Path to a multisample VCF file with needed scores.", exists=True, readable=True
     ),
-    target_sample: str = typer.Argument(..., help="Sample name.", exists=True, readable=True),
```

Comment:
Why are we removing these defaults?

Reply:
The `target_sample` here is a string here not a file path checking for for existence or readable is not relevant. Essentially, the functionality of this `vcf2adpc `is such that it pulls the information for the `target_sample `from the aggregated `bcf ` input file and writes to `adpc.bin` (outfile) for contamination workflow.